admin/app/components/certification-centers/invitations.hbs

```diff
@@ -4,7 +4,7 @@
   </header>
   <div class="content-text content-text--small">
     <div class="table-admin">
-      {{#if @certificationCenterInvitations}}
+      {{#if this.sortedInvitations}}
```

Comment:
**Question :**

Si dans le template on fait appel à `this.sortedInvitations`, est-ce Ember est assez smart pour faire du caching sur le retour de ce getter ?

Je pose cette question parce que l'on risque potentiellement de faire X fois le sorting d'un tableau en fonction du nombre d'utilisation de `this.sortedInvitations`.

Reply:
Autre question : l'api les renvoie triées par date et je vois pas de bouton d'action de tri du coup j'ai l'impression qu'on a pas besoin de faire un sorted

Reply:
Après vérification, ce tri avec `this.sortedInvitations` est effectué *à chaque rendu* de la liste des invitations 😦 donc pas de cache 😦

Cette utilisation de `this.sortedInvitations` a été mise en place ici car l'envoi d'une nouvelle invitation ajoute une nouvelle invitation au niveau du model Ember (`this.store.createRecord('certification-center-invitation' …`) et on veut que les invitations soient bien ordonnées dès cet ajout, sans avoir à recharger la page qui ramènerait les invitations triées par le serveur. Dans ces conditions effectivement le tri est fait des 2 coté : coté serveur pour l'affichage initial et par Ember pour chaque affichage et chaque fois qu'on ajoute une nouvelle invitation. Le tri est donc effectué à de nombreuses reprises de manière inutile 😦

Ce code a repris la logique utilisée dans le code des organizations.